source/speech/__init__.py

```diff
@@ -487,22 +487,22 @@ def splitTextIndentation(text):
 
 
 def getIndentationSpeech(indentation: str, formatConfig: Dict[str, bool]) -> SpeechSequence:
-	"""Retrieves the phrase to be spoken for a given string of indentation.
+	"""Retrieves the indentation speech sequence for a given string of indentation.
 	@param indentation: The string of indentation.
 	@param formatConfig: The configuration to use.
-	@return: The phrase to be spoken.

```

Comment:
Why was this deleted?

Reply:
It's already pretty clear from the first sentence of the doc string what this returns. It also no longer returns a phrase, and saying that this returns a speech sequence isn't necessary because the function is type annotated.